Base/res/ladybird/default-config/BrowserContentFilters.txt

```diff
@@ -63,22 +63,32 @@
 /unbxdAnalytics.js
 /v1/log_event
 /x.gif?
+12ezo5v60.com
 15gifts.com
+162.252.214.4
```

Comment:
Tracking random IP addresses doesn't seem to be very useful either? Those are easily changed.